src/main/java/com/revaturelabs/ask/question/QuestionJsonDeserializer.java

```
@@ -61,11 +64,24 @@ public Question deserialize(JsonParser p, DeserializationContext ctxt)
 
       if ((Integer) root.userId != null) {
         question.setQuestionerId(root.userId);
-      }
+      } else 
+        question.setQuestionerId(1);
```

remove this, let it fail if they don't give you a questioner id

---

setting it to 0 is ok, if you prefer